### PR TITLE
SISRP-15693 - Hide Study List / GTW Forms Links on Advising Resources Card till GL5.3

### PR DESCRIPTION
**Pull request by christianvuerings**
https://jira.berkeley.edu/browse/SISRP-15693

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -323,6 +323,8 @@ features:
   manage_site_mailing_lists: false
   cs_academic_planner: false
   cs_academic_progress_report: false
+  cs_advising_semester_study_list: false
+  cs_advising_gte_forms: false
```
Comment (johncrossman) [inline config/settings.yml]: Minor point: How about alpha-order like the others?

Comment (christianvuerings) [inline config/settings.yml]: @johncrossman yes! Just pushed up a fix

Also been thinking to do it for the entire list, since it's getting pretty long.

**Comment by christianvuerings** [comment]
1 green travis, restarted the other one.

**Comment by johncrossman** [comment]
One is all we need.

**Pull request merged**